src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/UsersController.cs

```diff
@@ -80,5 +81,28 @@ public async Task<ActionResult> CreateUserAsync(Guid organisationId, CreateUserR
 
             return Ok();
         }
+
+        [Route("users/{userId}")]
+        [HttpGet]
+        public async Task<ActionResult> GetUserById(string userId)
+        {
+            var user = await _usersRepository.GetUserById(userId);
+
+            if (user is null)
+            {
+                return NotFound();
+            }
+
+            var userViewModel = new GetUserViewModel()
+            {
+                Name = $"{user.FirstName} {user.LastName}",
+                PhoneNumber = user.PhoneNumber,
+                EmailAddress =  user.Email,
+                Disabled = user.Disabled,
+                PrimaryOrganisationId = user.PrimaryOrganisationId
+            };
+
+            return Ok(userViewModel);
```

Comment:
We're not actually returning a view here so I think the variable (and corresponding) type would be better without the `ViewModel` suffix.